packages/marko-web-native-x/templates/story.marko

```diff
@@ -0,0 +1,39 @@
+import { buildImgixUrl } from "@parameter1/base-cms-image";
```

Comment:
Remove this - the site should implement the template - there's no need for a basic template as it will never be used by an implementing website

Reply:
If anything, move this to the README as an example/documatation